portal/src/graphql/adminapi/UserDetailsScreen.module.css

```diff
@@ -2,8 +2,25 @@
   grid-column: 1 / span 8;
 
   @apply tablet:col-span-full;
+
+  @apply flex flex-col;
 }
 
 .siweEnabledTabWarningMessageBar {
   @apply mt-5;
 }
+
+.screenContent {
+  @apply flex-1-0-auto;
+
+  grid-template-rows: min-content minmax(min-content, 1fr);
+  grid-auto-columns: 1fr;
```

Comment:
It seems this `grid-auto-columns: 1fr;` has no effect, because `grid-template-columns: repeat(12, 80px);` already defines it has 12 columns, and 80px each.